src/business-logic/services/device-connection-service-implementation.ts

```diff
@@ -0,0 +1,100 @@
+import { DeviceConnectionService } from './interfaces/device-connection-service'
+import { Device } from '../../model/entities/device'
+import { DeviceConnectionFactory } from './interfaces/device-connection-factory'
+import { DeviceConnection } from './interfaces/inewsgateway-device-connection'
+
+type DeviceAggregate = {
+  deviceConnection: DeviceConnection,
+  device: Device
+}
+
+export class DeviceConnectionServiceImplementation implements DeviceConnectionService {
+  private readonly connectedDevices: Map<string, DeviceAggregate> = new Map()
+
+  constructor(private readonly deviceConnectionFactory: DeviceConnectionFactory) {
+  }
+
+  public async createConnection(device: Device): Promise<DeviceConnectionStatus> {
+    if (this.connectedDevices.has(device.id)) throw new DeviceAlreadyConnectedError(device.id)
+    const deviceConnection: DeviceConnection = this.deviceConnectionFactory.createDeviceConnection(device)
+
+    await deviceConnection.connect()
+
+    this.connectedDevices.set(device.id, {deviceConnection, device})
+
+    return DeviceConnectionStatus.CONNECTED
+  }
+
+  public async send(_deviceId: string, _params: string[]): Promise<void> {
+    const conDevice: DeviceAggregate | undefined = this.connectedDevices.get(_deviceId)
```

Comment:
I assume that this is a very deceiving device that we should hold an open eye on 😛 
It would be nice with the full name `connectedDevice` 🙏